src/main/java/com/cisco/trex/stateful/model/stats/PortHistData.java

```diff
@@ -0,0 +1,85 @@
+package com.cisco.trex.stateful.model.stats;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public class PortHistData {
+
+    @JsonProperty("cnt")
+    public Integer cnt;
+
+    @JsonProperty("high_cnt")
+    public Integer high_cnt;
+
+    @JsonProperty("histogram")
+    public List<HistogramData> histogram;
+
+    @JsonProperty("max_usec")
+    public Integer max_usec;
+
+    @JsonProperty("min_usec")
+    public Integer min_usec;
+
+    @JsonProperty("s_avg")
+    public Double s_avg;
+
+    @JsonProperty("s_max")
+    public Double s_max;
+
+    public Integer getCnt() {
```

Comment:
it's better to add @JsonProperty(key) annotation for all set/getXXX if it has corresponding field in json response.

Reply:
ok